### DISCUSSION TITLE
Visual Hint Glossary

### DISCUSSION THREAD

**Discussion by levone1** [jarun/nnn#1785] [Ideas]
Idea stemming from a question posted here - https://github.com/jarun/nnn/discussions/1784#

Maybe there are hardly any to explain, (maybe the three that are answered in my issue are all that there are), but maybe not...  I wonder if it would be good to put an entry in the docs grouping together the various visual elements in place by default, and explaining their meaning.  Maybe it could also include their location in the code, to be tweaked.

Thanks

**Comment by N-R-K** [comment]
I feel like most visual clues should be easy to figure out. And if there is some that isn't obvious, then maybe we should try to redesign to make it more obvious rather than adding a bunch of text to explain it.

But with that said, I'm not really opposed to have a page in the wiki explaining various visual clues if people find it useful.

**Comment by jarun** [comment]
I am closing this discussion since this is a very old thread and no one else seems interested. Probably because it doesn't take very long to get familiar with such things. Kind of learn on the go type stuff.

**Reply by levone1** [reply]
Forgot about it.  Thanks
